### DISCUSSION TITLE
how to use vuepress2.x

### DISCUSSION THREAD

**Discussion by moremind** [vuepress-theme-hope/vuepress-theme-hope#1353] [Q&A] [answered]
rt.

**Comment by Mister-Hope** [comment, marked as answer]
Meteorlxy is making breaking changes from time to time, and I have to change the code to make it compatible almost every 2 version, and it!s a huge waste of my time.(just see vp2 change log, you can see that 3 br has been made in the last 5 versions), so I won’t continue developing until vp2 gets a solid API(once it touch stable or rc)

**Reply by moremind** [reply]
yes,thanks for contributes.
